templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -13,7 +13,8 @@ contents:
     spec:
       initContainers:
       - name: discovery
-        image: "{{.Images.setupEtcdEnv}}"
+        image: "{{.Images.MachineConfigOperator}}"
+        command: ["/usr/bin/setup-etcd-environment"]
```

Comment:
going through some other changes, and noticed we need to take care of https://github.com/openshift/machine-config-operator/blob/master/pkg/operator/bootstrap.go#L104 as well?

CI seems to be unhappy about this change also (at least the last run)

Reply:
that change was going to be covered in https://github.com/openshift/machine-config-operator/pull/895 where i was removing all of the non-template refs to images. was on the fence about doing it here.  but you are probably right that i should. ill give it a try and see if that fixes ci

Reply:
after i first look thru the logs of course :)

Reply:
on second thought making the change in boostrap.go would be inconsistent with how I'm treating mcc/mcs/mcd up til now. #850 added the mco image and uses it in templates (like the change i have here) but doesn't remove any other references to the images throughout.

Reply:
don't we need https://github.com/openshift/machine-config-operator/blob/master/pkg/operator/bootstrap.go#L104 to actually _render_ the template for the etcd pod?

Reply:
guess not actually, so #895 it is

Reply:
I'd say let's move to https://github.com/openshift/machine-config-operator/pull/895 where this diff is there as well and work from there

Reply:
I'm also thinking this can be the issue actually, the MCO image isn't plumbed here at all today so this PR is running w/o the MCO image for setupEtcdEnv which relates to the whole convo in https://github.com/openshift/machine-config-operator/pull/895/files#r298159766

Reply:
the thing is that the other substitutions for the other images were in `manifests/` where this change is in templates so i was thinking that maybe that is the problem?

Reply:
i can't move this to 895 bc this change is needed for the installer pr. but can i pull that change from 895 and move it here?

Reply:
ok moved the 895 change to this pr so we'll see soon!